pkg/crc/preflight/preflight_linux.go

```diff
@@ -67,11 +67,13 @@ var libvirtPreflightChecks = [...]PreflightCheck{
 		flags:            SetupOnly,
 	},
 	{
-		configKeySuffix:  "check-crc-network",
-		checkDescription: "Checking if libvirt 'crc' network is available",
-		check:            checkLibvirtCrcNetworkAvailable,
-		fixDescription:   "Setting up libvirt 'crc' network",
-		fix:              fixLibvirtCrcNetworkAvailable,
+		configKeySuffix:    "check-crc-network",
```

Comment:
I'm not sure where to mention that the cleanup on linux is only partial for now? (no cleanup of libvirt group for example).

Reply:
How about putting that details in the documents?

Reply:
In my opinion, it's useful to have in the commit log too if someone looks at git history to try to understand why something is not part o f `cleanup`.

Reply:
@cfergeau sure I can add a line in the commit logs.